README.md

```diff
@@ -101,6 +101,13 @@ For example, to install dependencies on Arch Linux:
 sudo pacman -S --needed glib2 libgexiv2 meson vala gstreamer gst-plugins-base-libs gdk-pixbuf2 gobject-introspection gst-plugins-good gst-plugins-bad gst-plugin-va
 ```
 
+
+To install dependencies on Debian/Ubuntu:
+
+```bash
+apt install build-essential meson valac libgexiv2-dev libgstreamer1.0-dev libgdk-pixbuf-2.0-dev libgdk-pixbuf-2.0-0 gobject-introspection help2man valadoc
```

Comment:
Please pay attention to consistency. Some packages you have only listed `dev`, but `gdk-pixbuf` lists both `libgdk-pixbuf-2.0-dev` and `libgdk-pixbuf-2.0-0`.